hardkernel/odroid_h4/README.md

```diff
@@ -0,0 +1,35 @@
+# Hardkernel ODROID H4+ Blobs
+
+The directory contains blobs to produce a full Dasharo firmware image for
+Hardkernel ODROID H4+ platforms.
+
+Do not use or load software from this repository until you have carefully
+read the licenses assigned to the relevant components. By downloading or
+using the software component from this repository, you agree to the terms
+of license associated with the given software component. If you do not wish
+to so agree, do not install or use the software from this repository.
+Association of the software components and licenses is presented below.
+
+* `me.bin` - Intel Management Engine
+  * Version: v16.50.10.1351,
+  * License:  [PV Intel OBL Software License Agreement 11.2.2017][INTEL SLA].
+* `descriptor.bin` - Intel Flash Descriptor
+  * Version: v1.0,
+  * License:  [PV Intel OBL Software License Agreement 11.2.2017][INTEL SLA].
+
+Intel Management Engine and Flash Descriptor have been generated from Intel
+CSME for Gracemont v16.50.10.1351 software kit using genuine components provided by Intel, subject to the
```

Comment:
Gracemont is a core architecture. This should be Alder Lake-N. Also this lien needs wrapping.